justfile

```
@@ -0,0 +1,32 @@
+alias b := build
+alias c := check
+alias f := fmt
+alias t := test
+alias p := pre-push
+
+_default:
+  @just --list
+
+# Build the project
+build:
+   cargo build
+
+# Check code: formatting, compilation, linting, and commit signature
+check:
+   cargo +nightly fmt --all -- --check
+   cargo check --workspace --exclude 'examples' --all-features
```

I was thinking this needed to be a package specifier, not a path.
```suggestion
   cargo check --workspace --exclude 'example_*' --all-features
```

---

Same suggestion would apply to `just test`.

If we're following the `--exclude` pattern, then clippy would become

`cargo clippy --workspace --exclude 'example_*' --all-features --all-targets -- -D warnings`

---

My mistake, I though `--exclude` took a path and not a package.